### PR TITLE
Vmo 4145 unified blocks set group membership

### PR DESCRIPTION
**Pull request by safydy-r**
Before merge: 
- I will get answer for TODO & remove them

What I added:
- use a new the group membership component (no more dropdown selector, but with radio button shurtcuts)
- added output branching compornent

Note:
- setting exit.value when having Single exit mode will be done in vmo-4322
- in master, we already have setContactProp & Categorization

Screenshots
Screen Shot 2021-07-23 at 4 53 50 PM

### CODE REVIEW

File: src/components/interaction-designer/block-editors/GroupMembershipEditor.vue [hunk 1]
```diff
@@ -0,0 +1,164 @@
+<template>
```
Comment (safydy-r) [inline src/components/interaction-designer/block-editors/GroupMembershipEditor.vue:1]: I adopted the same approach from https://github.com/FLOIP/flow-builder/blob/master/src/components/interaction-designer/block-editors/ContactPropertyEditor.vue

File: src/components/interaction-designer/block-types/Core_SetGroupMembershipBlock.vue [hunk 2]
```diff
@@ -13,25 +13,13 @@
...
-            :show-labels="false"
-            :searchable="false" />
-        </div>
-      </validation-message>
-
-      <group-selector :block="block" />
+      <group-membership-editor :block="block" />
+
+      <hr>
+      <!--TODO: confirm the test in advanced branching `true` or expression from Single Exit mode eg: `block.value = 1`-->
```
Comment (safydy-r) [inline src/components/interaction-designer/block-types/Core_SetGroupMembershipBlock.vue:19]: I'll remove this before merge

File: src/components/interaction-designer/block-editors/GroupMembershipEditor.vue [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+    if (this.isMember === NULL_STRING_EXPRESSION) {
+      return this.MEMBERSHIP_ACTION.REMOVE
+    } else if (this.isMember === TRUTHY_STRING_EXPRESSION) {
+      return this.MEMBERSHIP_ACTION.ADD
+    }
+    return this.MEMBERSHIP_ACTION.SET_FROM_EXPRESSION
+  }
+
+  set membershipAction(value: string) {
+    console.log('membershipAction', value)
```
Comment (poojahpatil90) [inline src/components/interaction-designer/block-editors/GroupMembershipEditor.vue:124]: Can you remove the `console.log` ?

**Review by poojahpatil90 — APPROVED** [review]
@safydy-r Looks good to me. Just left a minor comment

File: src/components/interaction-designer/block-editors/GroupMembershipEditor.vue (hunk 3 above)
Comment (safydy-r) [reply src/components/interaction-designer/block-editors/GroupMembershipEditor.vue:124]: Thanks, good catch

**Pull request merged**